picard/config_upgrade.py

```diff
@@ -371,4 +415,5 @@ def upgrade_config(config):
     cfg.register_upgrade_hook(upgrade_to_v2_6_0_dev_1)
     cfg.register_upgrade_hook(upgrade_to_v2_6_0_beta_2)
     cfg.register_upgrade_hook(upgrade_to_v2_6_0_beta_3)
+    cfg.register_upgrade_hook(upgrade_to_v2_7_0_dev_1)
```

Comment:
That's not totally correct, because the hook will not run for people running current dev version (2.7.0.dev1).
We need to bump version to 2.7.0dev2 and name this upgrade hook `upgrade_to_v2_7_0_dev_2`

Reply:
I was wondering how best to handle that.  Will make the change.  Thanks.